src/cocotb/share/lib/simulator/simulatormodule.cpp

```diff
@@ -1284,8 +1289,11 @@ static PyMethodDef gpi_sim_hdl_methods[] = {
     {"get_range", (PyCFunction)get_range, METH_NOARGS,
      PyDoc_STR("get_range($self)\n"
                "--\n\n"
-               "get_range() -> Tuple[int, int]\n"
-               "Get the range of elements (tuple) contained in the handle.")},
+               "get_range() -> Tuple[int, int, int]\n"
+               "Get the range of elements (tuple) contained in the handle. "
+               "The first two elements of the tuple specify the left and "
+               "right bounds, while the third specifies the direction (`1` "
+               "for ascending, `-1` for descending, and `0` for undefined).")},
```

Comment:
```suggestion
               "right bounds, while the third specifies the direction (``1`` "
               "for ascending, ``-1`` for descending, and ``0`` for undefined).")},
```
Use monospace, not italics for ReST rendering (https://cocotb--4212.org.readthedocs.build/en/4212/library_reference.html#cocotb.simulator.gpi_sim_hdl.get_range)